source/game/system/map/MapdataCannonPoint.hh

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "game/system/map/MapdataAccessorBase.hh"
+
+#include <egg/math/Vector.hh>
+
+namespace System {
+class MapdataCannonPoint {
+public:
+    struct CannonParameter {
+        f32 speed;
+        f32 height;
+        f32 decelFactor;
+        f32 endDecel;
+    };
```

Comment:
This struct belongs in `KartMove.cc`. Because it's a struct that's only used locally within that TU, we move it into the source file instead of the header so nothing can interface with it.